src/cluster_legacy.c

```diff
@@ -495,6 +495,7 @@ int clusterLoadConfig(char *filename) {
         }
         *p = '\0';
         memcpy(n->ip,aux_argv[0],strlen(aux_argv[0])+1);
+        clearCachedClusterSlotsResponse();
```

Comment:
Are we sure this touch point is necessary? This is during the loading flow.

Reply:
The load happens only during initializing the cluster, I think we can remove this touchpoint.